5_data_munge.yml

```diff
@@ -84,3 +89,16 @@ targets:
     command: summarize_all_dat(
       in_ind = '5_data_munge/out/daily_temperatures.rds.ind',
       out_file = target_name)
+  
+  # qaqc the daily temperature data.    
+  5_data_munge/out/daily_temperatures_qaqc.rds.ind:
+    command: qaqc_daily_temp_site_data(
+      temp_in_ind = '5_data_munge/out/daily_temperatures.rds.ind', 
+      site_in_ind = '5_data_munge/out/all_sites.rds.ind',
+      out_ind = target_name)
+      
+  5_data_munge/out/daily_temperatures_qaqc_summary.csv:
+    command: summary_qaqc_daily_temp_site('5_data_munge/out/daily_temperatures_qaqc.rds.ind',
+    out_file = target_name)
```

Comment:
This indentation on line 102 will cause an error -- needs to be formatted like other targets. It's hard to fix when you're not actually running these targets.